### ISSUE TITLE
[bug] Runtime error when batch get from a table with multiple pks

### ISSUE THREAD

**Issue by hou8** [edragonconnect/ecto_tablestore#36]
```elixir
test "repo - batch_get from table with multiple pks" do
  saved_orders  =
    Enum.reduce(1..3, [], fn var, cur_orders ->
      order = %Order{id: "#{var}", desc: "desc#{var}", num: var, price: 1.8 * var}
      {:ok, saved_order} =
        TestRepo.insert(order, condition: condition(:ignore), return_type: :pk)
      cur_orders ++ [saved_order]
    end)

  requests1 = [
    [
      %Order{id: "1", internal_id: Enum.at(saved_orders, 0).internal_id},
      %Order{id: "2", internal_id: Enum.at(saved_orders, 1).internal_id}
    ]
  ]

  requests2 = [
    {Order, [
      [{"id", "1"}, {"internal_id", Enum.at(saved_orders, 0).internal_id}],
      [{"id", "2"}, {"internal_id", Enum.at(saved_orders, 1).internal_id}]
    ]}
  ]

  {:ok, result} = TestRepo.batch_get(requests1)
  {:ok, ^result} = TestRepo.batch_get(requests2)

  [{Order, query_orders}] = result

  assert length(query_orders) == 2

  for order <- saved_orders do
    TestRepo.delete(%Order{id: order.id, internal_id: order.internal_id},
      condition: condition(:expect_exist)
    )
  end
end
```

1. `Order` has two pks, `id` and `internal_id`
2. In the above test case, `request1` could get result as expected, but `request2` will get a runtime error

```
10:53:07.874 [error] Task #PID<0.322.0> started from #PID<0.313.0> terminating
** (ExAliyunOts.RuntimeError) Error: "Unsupported primary key for value: nil"
    (ex_aliyun_ots 0.14.4) lib/ex_aliyun_ots/plainbuffer/plainbuffer.ex:297: ExAliyunOts.PlainBuffer.do_process_primary_key_value/2
    (ex_aliyun_ots 0.14.4) lib/ex_aliyun_ots/plainbuffer/plainbuffer.ex:193: ExAliyunOts.PlainBuffer.primary_key_column/2
    (elixir 1.12.2) lib/enum.ex:2385: Enum."-reduce/3-lists^foldl/2-0-"/3
    (ex_aliyun_ots 0.14.4) lib/ex_aliyun_ots/plainbuffer/plainbuffer.ex:141: ExAliyunOts.PlainBuffer.do_primary_keys/3
    (ex_aliyun_ots 0.14.4) lib/ex_aliyun_ots/plainbuffer/plainbuffer.ex:52: ExAliyunOts.PlainBuffer.serialize_primary_keys/1
    (ex_aliyun_ots 0.14.4) lib/ex_aliyun_ots/client/row.ex:267: ExAliyunOts.Client.Row.pks_to_batch_get_row/1
    (ex_aliyun_ots 0.14.4) lib/ex_aliyun_ots/client/row.ex:241: ExAliyunOts.Client.Row.do_request_to_batch_get_row/1
    (elixir 1.12.2) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (elixir 1.12.2) lib/task/supervised.ex:35: Task.Supervised.reply/5
    (stdlib 3.15.2) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: &:erlang.apply/2
    Args: [#Function<17.93583854/1 in ExAliyunOts.Client.Row.request_to_batch_get_row/1>, [%ExAliyunOts.Var.GetRow{columns_to_get: ["name", "desc", "num", "success?", "price", "lock_version"], end_column: nil, filter: nil, max_versions: 1, primary_keys: [[{"id", nil}, {"internal_id", nil}]], start_column: nil, table_name: "ecto_ots_test_order", time_range: nil, transaction_id: nil}]]
```

---

I'll keep working on this issue.

**Comment by hou8** [comment]
@xinz I have fixed this by #37

**Comment by xinz** [comment]
Thanks!
